aten/src/ATen/native/sparse/cuda/cuSPARSELtOps.h

```diff
@@ -30,7 +30,7 @@ TORCH_CUDA_CPP_API std::tuple<at::Tensor, int64_t, int64_t, bool, int64_t> _cslt
     bool transpose_result,
     int alg_id,
     int split_k,
-    bool split_k_one_kernel,
+    int split_k_mode,
```

Comment:
Looks like the `tuple` return type here needs to also be changed to `std::tuple<at::Tensor, int64_t, int64_t, int64_t, int64_t> `